tracing-opentelemetry-kafka/src/main/java/io/micronaut/tracing/opentelemetry/instrument/kafka/KafkaTelemetryHelper.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017-2023 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.tracing.opentelemetry.instrument.kafka;
+
+import java.util.Collection;
+
+import io.micronaut.context.ApplicationContext;
+import io.micronaut.context.annotation.Context;
+import io.opentelemetry.api.OpenTelemetry;
+
+/**
+ * Helper-class to access to kafkaTelemetry object from non-micronaut components.
+ *
+ * @since 4.6.0
+ */
+@Context
+public final class KafkaTelemetryHelper {
+
+    private static KafkaTelemetry kafkaTelemetry;
+
+    @SuppressWarnings("rawtypes")
+    KafkaTelemetryHelper(OpenTelemetry openTelemetry, KafkaTelemetryConfiguration kafkaTelemetryConfiguration,
+                         ApplicationContext applicationContext) {
+        Collection<KafkaTelemetryConsumerTracingFilter> consumerTracingFilters = applicationContext.getBeansOfType(KafkaTelemetryConsumerTracingFilter.class);
+        Collection<KafkaTelemetryProducerTracingFilter> producerTracingFilters = applicationContext.getBeansOfType(KafkaTelemetryProducerTracingFilter.class);
+        kafkaTelemetry = KafkaTelemetry.create(openTelemetry, kafkaTelemetryConfiguration, consumerTracingFilters, producerTracingFilters);
+    }
+
+    public static KafkaTelemetry getKafkaTelemetry() {
```

Comment:
why is this static state necessary?

Reply:
I did not find another way to use the KafkaTelemetry bean outside of the contest, i.e. from TracingConsumerInterceptor and TracingProducerInterceptor.

I connected them through the kafka configuration, like this:
```yaml
kafka:
  consumers:
    default:
      interceptor.classes: io.micronaut.tracing.opentelemetry.instrument.kafka.TracingConsumerInterceptor
  producers:
    default:
      interceptor.classes: io.micronaut.tracing.opentelemetry.instrument.kafka.TracingProducerInterceptor
```

If you mean another more correct way, please tell me and I'll fix it.